backend/src/main/java/com/htecgroup/skynest/repository/custom/impl/CassandraSaveWithTtlImpl.java

```diff
@@ -0,0 +1,25 @@
+package com.htecgroup.skynest.repository.custom.impl;
+
+import com.htecgroup.skynest.repository.custom.CassandraSaveWithTtl;
+import lombok.AllArgsConstructor;
+import org.springframework.data.cassandra.core.CassandraOperations;
+import org.springframework.data.cassandra.core.InsertOptions;
+import org.springframework.util.Assert;
+
+@AllArgsConstructor
+public class CassandraSaveWithTtlImpl<T> implements CassandraSaveWithTtl<T> {
+
+  private CassandraOperations operations;
+
+  @Override
+  public <S extends T> S save(S entity, int timeToLiveInSeconds) {
```

Comment:
What if I want this object to be persisted without timeToLive, meaning for good or until I explicitly request to delete it? Can we provide that option as well since you already used this generic approach ?

Reply:
InvalidJwtRepository also extends CassandraRepository, which already has a regular <code>save()</code> that persists indefinitely.